plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotConnectorSmokeTest.java

```diff
@@ -1554,23 +1554,23 @@ WHERE string_col IN ('null', 'array_null')"""))
         assertThat(query("""
                 SELECT COUNT(*)
                 FROM alltypes
-                WHERE long_col IS NULL"""))
-                .matches("VALUES (BIGINT '0')")
-                .isNotFullyPushedDown(FilterNode.class);
+                WHERE long_col IS NULL OR long_col = 0"""))
+                .matches("VALUES (BIGINT '2')")
+                .isFullyPushedDown();
 
         assertThat(query("""
                 SELECT COUNT(*)
                 FROM alltypes
-                WHERE long_col IS NOT NULL"""))
-                .matches("VALUES (BIGINT '11')")
-                .isNotFullyPushedDown(FilterNode.class);
+                WHERE long_col IS NOT NULL AND long_col != 0"""))
```

Comment:
Same here, maybe have an explicit test for null handling enabled vs disabled in a table w the same data, and check for MIN_VALUE and/or the default null value. lmk, can talk offline about it when you have some time.